run_integration_tests

```diff
@@ -3,6 +3,6 @@
 export $(cat .env | xargs)
 export INTEGRATION=1
 # Start test
-npx jest --silent=true --coverage -c tests/integration_tests/jest.config.js --detectOpenHandles
+npx jest --silent=true --coverage -c tests/integration_tests/jest.config.js #--detectOpenHandles tests/integration_tests/users.test.ts
```

Comment:
since commented out you can discard this change
it's just part of regular integration tests right?

Reply:
yes this change can be discarded